docs/UserGuide.md

```diff
@@ -299,10 +307,32 @@ If you want a better score you can always reattempt a quiz!
 
 ## 4. FAQ
 
+**Q**: Am I be able to re-attempt the same question after my quiz ends?
+
+**A**: Yes, you may. Questions for the quizzes are chosen randomly from all of the questions in a certain topic. Hence, there is a chance that the question you have attempted before gets chosen again if you choose to attempt another quiz from the same topic.
+
+**Q**: Why does my stats only improve as I do more quizzes? Shouldn't the penalities from my previous quizzes remain?
+
+**A**: E-Duke-8 tries to motivate you to understand CS2113/T concepts better. E-Duke-8 also encourages learning from failure. Therefore, you do not have to be disheartened by incorrect answers when attempting quizzes. Learn from the explanations given and get the question correct in your future quiz attempts to improve your stats!
+A correct answer in a future quiz attempt would also mean that you have come to understand the concept being tested in the question, and hence, you should deserve the points for it.  
+
+**Q**: Why does my hint usage indicator for a question not disappear after I re-attempt the question without using hint?
+
+**A**: This is because the hint usage serves as an indicator that you might still have some doubts in the topic, and hence you might want to read more about it. Resetting the count for hint usage might take away this benefit. 
+
+**Q**: Using hints will limit the total points that I can earn to build up my stats, so why should I still use hint?
+
+**A**: While points you earn will indeed be lesser, you are still encouraged to use hint when answering questions to spur your thinking, or to confirm your thoughts. 
+       Hints might give you an alternative perspective to think about the question at hand, which may be helpful to how you answer similar questions in the future.
+
+**Q**: How are the points awarded for correct answers?
+
+**A**: A correct answer without requesting for a hint will earn you **2 points**, whereas a correct answer with hint requested will earn you **1 point**. You will not be awarded any points for incorrect answers.
+
```

Comment:
This is really detailed, and addresses very realistic issues! Thanks for doing this!!